facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -3,79 +3,112 @@
 @import layout.EditionalisedLink
 @import model.EmailAddons.EmailContentType
 
+@import layout.ItemClasses
+@import layout.ContentCard
+@import views.support.RemoveOuterParaHtml
 @(page: model.PressedPage)(implicit request: RequestHeader)
 
 @import views.support.EmailHelpers._
-@import views.support.TrailCssClasses.toneClass
+@import views.support.TrailCssClasses.toneClassFromStyle
 @import model.pressed.{Feature, ExternalLink}
 @import implicits.Requests.RichRequestHeader
 @import implicits.ItemKickerImplicits._
 
 
-@headline(pressedContent: PressedContent) = {
+@headline(card: ContentCard) = {
     <h3 class="headline">
-        @defining(FaciaCardHeader.fromTrail(pressedContent, None)) { header =>
-            @header.kicker.map { kicker =>
-                <span class="fc-item__kicker">@Html(kicker.kickerHtml)</span>
-                <span class="kicker-separator">/</span>
-            }
+        @card.header.kicker.map { kicker =>
+            <span class="fc-item__kicker">@Html(kicker.kickerHtml)</span>
+            <span class="kicker-separator">/</span>
+        }
 
-            @defining(if (request.isEmailConnectedStyle) "-connected" else "") { suffix =>
-                @if(header.isGallery) { @icon("gallery" + suffix) }
-                @if(header.isAudio) { @icon("podcast" + suffix) }
-                @if(header.isVideo) { @icon("video" + suffix) }
+        @defining(if(request.isEmailConnectedStyle) "-connected" else "") { suffix =>
+            @if(card.header.isGallery) {
```

Comment:
I actually find the other way easier to scan, but total personal preference as we don't have code style for this I don't think